features/initial-letter.yml

```
@@ -0,0 +1,6 @@
+name: initial-letter
+description: The `initial letter` CSS property sets the number of lines the first letter occupies.
```

Huh, I didn't know this existed! I'm making some suggestions here to make it more clear where this applies and to work in some related typography terms, though they're not strictly required. What do you think?

```suggestion
description: The `initial-letter` CSS property sets the number of lines the first letter of an element occupies. You can use the property to make a raised capital or drop cap.
```